int32 upperLeft
int32 lowerRight
string color
string cameraID